pontoon/base/urls.py

```diff
@@ -30,6 +30,15 @@
     url(r'^contributor/(?P<email>[\w.%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,4})/$',
         RedirectView.as_view(url="/contributors/%(email)s/", permanent=True)),
 
+    # List all active localization teams
+    url(r'^teams/$',
+        views.locales,
+        name='pontoon.teams'),
+
+    # Redirect to a team page
+    url(r'^teams/(?P<locale>[A-Za-z0-9\-\@\.]+)/$',
```

Comment:
Why this other URL? I don't see it being used anywhere.

If there's some JS that auto-generates links that end up looking like this I think we should change the JS instead of accommodating it.